mapdata/WarcraftLegacies/AbilityData/Core/961760339.json

```diff
@@ -0,0 +1,141 @@
+{
+  "OldId": 1718832961,
+  "NewId": 961760339,
```

Comment:
This ability (Summon Treants) doesn't have enough levels. You can research Nature's Blessing 3 times which gets the ability to level 4. But doing so actually stops the ability from doing anything, since it caps at level 3.